### PR TITLE
Implement RPC file transfer

### PR DESCRIPTION
**Pull request by McSinyx**

### CODE REVIEW

**Comment by Huy-Ngo** [comment]
Shouldn't we name the folder by the practical work number?

File: rpc/server.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+from xmlrpc.client import Binary
+from xmlrpc.server import SimpleXMLRPCServer
+
+path = Path(__file__).parent.parent / 'images' / 'file-protocol-1.pdf'
```
Comment (Huy-Ngo) [inline rpc/server.py]: Why do you need to open `file-protocol-1.pdf`?

Comment (Huy-Ngo) [reply rpc/server.py]: Hmm apparently you send it to client with name "logo"?

Comment (McSinyx) [reply rpc/server.py]: LOL good catch!  Edit: fixed.

**Comment by McSinyx** [comment]
> Shouldn't we name the folder by the practical work number?

The instructions said to

> Copy your TCP file transfer system to a new directory called RPC

And I named it `rpc` which is as close as a non-psychopath can get to path naming :smile:

**Review by Huy-Ngo — APPROVED** [review]

**Comment by Huy-Ngo** [comment]
GH doesn't count Python somehow.

**Pull request merged**